variables.tf

```diff
@@ -74,12 +74,11 @@ variable "enable_private_default_route" {
 
 variable "flow_logs" {
   type = object({
-    iam_role_name     = string
+    iam_role_name            = string
     iam_role_boundary_policy = string
```

Comment:
Nice... Last comment... This list is sorted alphabetically, so please keep it that way.

Reply:
Done